### DISCUSSION TITLE
Hi, I would like to ask how your "Copilot for Xcode" was packaged?

### DISCUSSION THREAD

**Discussion by cloudmoonocus** [community/community#145773] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

When I directly archive it, I cannot start the Extension Service Target, so I would like to ask how you package it?

**Comment by devm33** [comment, marked as answer]
👋 Hi @cloudmoonocus there's some documentation for this here: https://github.com/github/CopilotForXcode/blob/main/DEVELOPMENT.md See also https://github.com/github/CopilotForXcode/issues/37
